todo_app/app.py

```diff
@@ -8,7 +9,30 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session_items.get_items()
+    sorted_items = sorted(items, key=lambda i: i['status'], reverse=True)
+    return render_template('index.html', items=sorted_items)
+
+
+@app.route('/addItem', methods=['POST'])
+def add_items():
+    new_item = request.form.get('todoitem')
+    session_items.add_item(new_item)
+    return index()
```

Comment:
This would be nicer returning a redirect rather than the index directly - currently if I add an item, then the URL still records that it is at /addItem, and if the user refreshes then it will post the item a second time. (See ["Redirects and Errors"](https://flask.palletsprojects.com/en/1.1.x/quickstart/))